articles/iot-edge/tutorial-nested-iot-edge.md

```diff
@@ -589,6 +629,22 @@ Notice that the image URI that we used for the simulated temperature sensor modu
 
 On the device details page for your lower layer IoT Edge device, you should now see the temperature sensor module listed along the system modules as **Specified in deployment**. It may take a few minutes for the device to receive its new deployment, request the container image, and start the module. Refresh the page until you see the temperature sensor module listed as **Reported by device**.
 
+## IotEdge check
+Run the iotedge check command to verify the configuration and troubleshoot errors.
+Iotedge check can be performed in nested hierarchy, even if the children doesn't have direct internet access.
+When running "iotedge check" from the lower layer the program will try to pull the image from the parent through port 443.
+In this How-to, we use port 8000, so we need to specify it:
```

Comment:
One last clarification -- this line should refer to 'this tutorial' not 'this how-to'

Reply:
fixed!